backend/contracts/Habit.sol

```diff
@@ -12,15 +12,15 @@ contract Habit {
     // user addr -> User struct & attributes
     // Mapping users address -> user
     mapping(address => user) users;
-    address owner;
+    address owner = msg.sender;
     uint start_time;
     uint end_time;
     uint256 pool;
 
 
     // Create the challenge and set a date range
     // client side will convert date/time to block.timestamp offset
-    constructor(uint start_time_) public payable {
```

Comment:
I'll be changing this to reflect the Dice a bit more, without constructor and having a mapping of habits instead.